modules/op-signing-secrets-in-tekton-chains.adoc

```diff
@@ -39,6 +39,54 @@ Cosign prompts you for a password, and creates a Kubernetes secret.
 
 . Store the encrypted `cosign.key` private key and the `cosign.password` decryption password in the `signing-secrets` Kubernetes secret. Ensure that the private key is stored as an encrypted PEM file of the `ENCRYPTED COSIGN PRIVATE KEY` type.
```

Comment:
> Ensure that the private key is stored as an encrypted PEM file of the `ENCRYPTED COSIGN PRIVATE KEY` type.

1. Why would customers care about it? It's responsibility of `cosign` to store the correct thing.
2. Cosign 2.0.2 stores file of type `ENCRYPTED SIGSTORE PRIVATE KEY`

Reply:
@PuneetPunamiya could you please comment whether I should remove the "ensure..." phrase?

Reply:
Good catch @ppitonak, yeah we should remove this point 👍🏻

Reply:
Done.

Reply:
I still see this in code and preview :/